tools/ui-components/src/image/image.types.tsx

```diff
@@ -0,0 +1,5 @@
+export interface ImageProps {
+  alt: string;
```

Comment:
Technically we can omit `alt` if the image is purely for decoration but let's keep it required for now to promote the accessibility practice. We can revisit this later.

Reply:
I think, if it's decorative, we want `alt=""`, rather than omitting it.  [This](https://www.w3.org/WAI/tutorials/images/decorative/) is what w3 has to say.

Reply:
Ah I was thinking about making this prop optional and default to an empty string if it is not defined. I guess it would be a bit more convenient for the components that use this one. On the other hand, we can easily have our images inaccessible if we forget to pass `alt` to the component.

Reply:
Oh, sorry, yeah defaulting to an empty string sounds good!

Reply:
No, sorry, now that I re-read my comment, I wasn't being clear at all. Thanks for making me clarify it 😄